.github/issue_template.md

```diff
@@ -12,12 +12,18 @@ INSERT CODE HERE
 ```
 
 #### System information
-* Please report:
-1. RAVEN version (stabile [release](https://github.com/SysBioChalmers/RAVEN/releases), `devel` branch?)
-2. Operating system (Windows/Mac/Linux; include version)
+<!-- Please report the complete output from checkInstallation, for instance:
+ > Installing from location             C:\Work\GitHub\RAVEN
+ > Checking RAVEN release               2.7.12
+   > You are running the latest RAVEN release
+ > Checking MATLAB release              2022a
+ > Checking system architecture         win64
+ > Set RAVEN in MATLAB path             Pass
+ > Save MATLAB path                     Pass
+
+etc. etc.
+ -->
 
 **I hereby confirm that I have:**
 <!-- Note: replace [ ] with [X] to check the box -->
-- [ ] Followed the [guidelines](https://github.com/SysBioChalmers/RAVEN/wiki/Installation) to install RAVEN.
 - [ ] Checked that a similar issue does not [already exist](https://github.com/SysBioChalmers/RAVEN/issues?utf8=%E2%9C%93&q=is%3Aissue)
-- [ ] If suitable, needed, asked first in the [Gitter chat room](https://gitter.im/SysBioChalmers/RAVEN) about the issue
```

Comment:
I was hesitant about this, but I think it makes sense for people to write up issues here rather than push them on Gitter where the same issue might be raised again and again.